src/Nox.Types.EntityFramework/Abstractions/NoxDatabaseConfigurator.cs

```diff
@@ -14,13 +14,14 @@ public abstract class NoxDatabaseConfigurator : INoxDatabaseConfigurator
                 { NoxType.Text, new TextDatabaseConfigurator() },
                 { NoxType.Number, new NumberDatabaseConfigurator() },
                 { NoxType.Money, new MoneyDatabaseConfigurator() },
-                { NoxType.CountryCode2, new CountryCode2Configurator() }
+                { NoxType.CountryCode2, new CountryCode2Configurator() },
```

Comment:
I see that all other configurator have "Database" in their name. Should the name here be CountryCode2DatabaseConfigurator, for consistency?